docs/griptape-cloud/terms-of-service.md

```diff
@@ -1,74 +1,391 @@
+**GRIPTAPE CLOUD TERMS OF SERVICE**
+
+**LAST UPDATED: FEBRUARY 14, 2024**
+
+**BY CLICKING** "**I ACCEPT,**"**ACCESSING OUR USING GRIPTAPE INC'S
+("GRIPTAPE"OR "WE" SERVICE**, YOU ("CUSTOMER"OR "YOU" AGREE THAT YOU
+HAVE READ AND UNDERSTOOD, AND, AS A CONDITION TO ITS ACCESS TO OR USE OF
+THE SERVICE, YOU AGREE TO BE BOUND BY, THE FOLLOWING TERMS, INCLUDING
+GRIPTAPE'S PRIVACY POLICY (TOGETHER, THIS "**AGREEMENT**". IF YOU DO NOT
+AGREE TO THE TERMS AND CONDITIONS SET FORTH HEREIN, THEN YOU DO NOT HAVE
+GRIPTAPE'S PERMISSION TO ACCESS OR USE THE SERVICE. YOUR USE OF THE
+SERVICE, AND GRIPTAPE'S MAKING AVAILABLE THE SERVICE TO YOU, CONSTITUTES
+AN AGREEMENT BY GRIPTAPE AND BY YOU TO BE BOUND BY THIS AGREEMENT. THE
+EFFECTIVE DATE OF THIS AGREEMENT IS THE DATE THAT YOU ACCEPT OR AGREE TO
+THIS AGREEMENT BY CLICKING "I ACCEPT,"OR BY ACCESSING OR USING THE
+SERVICE ("**EFFECTIVE DATE**". If Customer is an entity, organization,
+or Griptape, then: (i) the individual accepting this Agreement on
+Customer's behalf represents and warrants that they have authority to
+bind Customer to this Agreement; (ii) Customer agrees to be bound by
+this Agreement; and (iii) the term \"**Customer**\" shall refer to such
+entity.
+
+1.  **USE OF THE SERVICE**.** **Griptape offers an online,
+    software-as-a-service platform, where Griptape hosts software
+    applications ("**Applications**" and enables Customers to deploy,
+    run, and manage their Applications on the platform ("**Griptape**"
+    "**Griptape Cloud**" or "**Service**". Applications may include
+    those written using the Griptape framework, which is comprised of
+    open source software development tools for interfacing with
+    artificial intelligence (AI) systems such as large language models
+    (LLMs) and executing other business logic. The Griptape framework is
+    available at and is licensed for use by Customer under an open
+    source license ("**Griptape Framework**". As part of the Service,
+    Griptape may make its API** **(as defined below) and SDK (as defined
+    below) available to Customer.** **The Service, API, and SDK are
+    collectively referred to herein as the "**Griptape IPR**"and any
+    references to the Griptape IPR includes all components and portions
+    thereof. Subject the terms and conditions of this Agreement,
+    Griptape will provide Customer with access the Service using access
+    credentials provided by Griptape to permit Customer ("**Access
+    Credentials**" to access and use the Service. Customer will provide,
```

Comment:
Caught my own typo, this parenthetical string is not needed at all. Can cut parentheses and all chars between them.